appeals/api/src/database/seed/data-samples.js

```diff
@@ -151,70 +151,71 @@ export const localPlanningDepartmentList = [
 	}
 ];
 
+/** @type {AppealSite[]} */
```

Comment:
I was a bit confused of where the case has changed, as in the DB is `postcode` and I don't think has changed recently. But I see you're mapping it below.